docker/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Mainflux
+# SPDX-License-Identifier: Apache-2.0
+
+version: "3.7"
+
```

Comment:
Do we need this?

Reply:
Agent and Export will be moving to this repo so yes